### DISCUSSION TITLE
How to render Title attribute dynamically

### DISCUSSION THREAD

**Discussion by alphayowakarindi** [livewire/livewire#8589] [1. Help] [answered]
Below is my `ShowPost` component and I'd like the `Title` attribute to be dynamic depending on the `post title` instead of hard coding `'Laravel Roadmap'`. Passing `'title' => $post->title` in the blade template it doesn't work too. How can I make this dynamic?

    <?php
    
    namespace App\Livewire\Posts;
    
    use App\Models\Post;
    use Livewire\Attributes\Title;
    use Livewire\Component;
    
    class ShowPost extends Component
    {
        public $post;
    
        public function mount(Post $post)
        {
            $this->post = $post;
        }
    
        #[Title('Laravel Roadmap')]
        public function render()
        {
            return view('livewire.posts.show-post', [
                'post' => $this->post,
            ]);
        }
    }

**Comment by alphayowakarindi** [comment, marked as answer]
We can use the `title()` method chained to the `view` helper. [Reference](https://livewire.laravel.com/docs/components#setting-the-page-title:~:text=If%20you%20need%20to%20pass%20a%20dynamic%20title%2C%20such%20as%20a%20title%20that%20uses%20a%20component%20property%2C%20you%20can%20use%20the%20%2D%3Etitle()%20fluent%20method%20in%20the%20component%27s%20render()%20method%3A) from the docs.

  ```public function render()
    {
        return view('livewire.posts.show-post', [
            'post' => $this->post,
        ])->title($this->post->title);
    }

**Reply by DanielUgalde95** [reply]
if you use #[Lazy()) not work
